### PR TITLE
Add Russian localization

### PR DESCRIPTION
**Pull request by PinkGoosik**

### CODE REVIEW

**Review by DoctorNight799 — APPROVED** [review]

**Review by paulevsGitch — APPROVED** [review]

**Review by paulevsGitch — APPROVED** [review]

**Comment by paulevsGitch** [comment]
Now translation is a bit better

**Review by Spusik — APPROVED** [review]

File: src/main/resources/assets/trinkets/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "trinkets.tooltip.slots.any": "Можно надеть в §9любой§r слот для аксессуаров",
+  "trinkets.tooltip.slots.list": "Можно надеть в слоты для аксессуаров:",
+  "trinkets.tooltip.slots.single": "Можно надеть в слот для аксессуаров %d",
+  "trinkets.tooltip.attributes.all": "Когда надето:",
+  "trinkets.tooltip.attributes.single": "Когда надето в слот для аксессуаров %d:",
+  "trinkets.tooltip.attributes.slots": "%d Слотов",
```
Comment (Brawaru) [inline src/main/resources/assets/trinkets/lang/ru_ru.json:7]: I don't think capitalisation is required here in Russian.

**Review by BasiqueEvangelist — APPROVED** [review]

**Review by COBASilver — APPROVED** [review]

File: src/main/resources/assets/trinkets/lang/ru_ru.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "trinkets.slot.chest.necklace": "Ожерелья",
+  "trinkets.slot.chest.back": "Спины",
+  "trinkets.slot.chest.cape": "Плаща",
+  "trinkets.slot.legs.belt": "Пояса",
+  "trinkets.slot.feet.aglet": "Эглета",
+  "trinkets.slot.feet.shoes": "Обуви",
+  "trinkets.slot.hand.glove": "Перчатки",
+  "trinkets.slot.offhand.glove": "Перчатки",
+  "trinkets.slot.hand.ring": "Кольца",
+  "trinkets.slot.offhand.ring": "Кольца"
```
Comment (Brawaru) [inline src/main/resources/assets/trinkets/lang/ru_ru.json:19]: Just making sure, these are not used anywhere else besides `trinkets.tooltip.slots.single`? If so, they also probably shouldn't be capitalised.

**Review by Brawaru — APPROVED** [review]
Looks alright.

**Review by aSomik — APPROVED** [review]

**Review by DeelTer — APPROVED** [review]
cool.

File: src/main/resources/assets/trinkets/lang/ru_ru.json (hunk 2 above)
Comment (PinkGoosik) [reply src/main/resources/assets/trinkets/lang/ru_ru.json]: @emilyploszaj Are those used anywhere else besides tooltips? Im not sure

**Review by V972 — APPROVED** [review]

**Comment by Felix14-v2** [comment]
@emilyploszaj sorry for ping, but during [completely rewrite](https://github.com/PinkGoosik/trinkets/pull/1) the translation appearance we came across the line `trinkets.tooltip.attributes.slots` – we don't know what circumstances it is used. Could you clarify this?

File: src/main/resources/assets/trinkets/lang/ru_ru.json (hunk 2 above)
Comment (emilyploszaj) [reply src/main/resources/assets/trinkets/lang/ru_ru.json]: These are the canonical "proper noun" names of slots, and currently I believe they're only used in tooltips. They're capitalized in english

**Comment by emilyploszaj** [comment]
> @emilyploszaj sorry for ping, but during [rewrite](https://github.com/PinkGoosik/trinkets/pull/1) the translation appearance we came across the line `trinkets.tooltip.attributes.slots` – we don't know what circumstances it is used. Could you clarify this?

Took me a second to figure out, it's the translation key for attributes, something like "+4 Ring Slots"

**Comment by PinkGoosik** [comment]
> Took me a second to figure out, it's the translation key for attributes, something like "+4 Ring Slots"

So in this case "%d" is "+4 Ring"? Can you please separate it to number and slot name? Would aid a lot

**Comment by Felix14-v2** [comment]
> translation key for attributes, something like "+4 Ring Slots"

That is, in trinkets there is an opportunity to modify the number of trinket-slots through attributes?

**Comment by emilyploszaj** [comment]
> > Took me a second to figure out, it's the translation key for attributes, something like "+4 Ring Slots"
> 
> So in this case "%d" is "+4 Ring"? Can you please separate it to number and slot name? Would aid a lot

Slot name is pulled from the translation keys here, and the number formatting is a vanilla translation key.

**Comment by PinkGoosik** [comment]
@emilyploszaj ready to be merged

**Comment by emilyploszaj** [comment]
Sorry this has taken so long to look at properly, why are the glove and ring slots notated as separate from their standard ones? Trinekts handles slots with the same translation by merging info if applicable, and I think it'd be ideal they be named the same thing. Other than that, this looks ready to be merged.

**Comment by PinkGoosik** [comment]
Done

**Pull request merged**